From d1a92a4d4da80b4cdc53164a42cbe29c03b1c8c1 Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Tue, 7 Dec 2010 14:12:55 -0600
Subject: [PATCH] libnm-glib: actually define the "ip-interface" property

Looks like it got left out of 6ab5a9597a26235c9839df346fc3ce95f35a240f
---
 libnm-glib/nm-device.c |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/libnm-glib/nm-device.c b/libnm-glib/nm-device.c
index ea2a91d..b8998f7 100644
--- a/libnm-glib/nm-device.c
+++ b/libnm-glib/nm-device.c
@@ -477,6 +477,20 @@ nm_device_class_init (NMDeviceClass *device_class)
 						  G_PARAM_READABLE));
 
 	/**
+	 * NMDevice:ip-interface:
+	 *
+	 * The IP interface of the device which should be used for all IP-related
+	 * operations like addressing and routing.
+	 **/
+	g_object_class_install_property
+		(object_class, PROP_IP_INTERFACE,
+		 g_param_spec_string (NM_DEVICE_IP_INTERFACE,
+						  "IP Interface",
+						  "IP Interface name",
+						  NULL,
+						  G_PARAM_READABLE));
+
+	/**
 	 * NMDevice:udi:
 	 *
 	 * The Unique Device Identifier of the device.
-- 
1.7.3.4

